### DISCUSSION TITLE
PlotBrowserTrack – restrict groups for useMatrix panel

### DISCUSSION THREAD

**Discussion by SianPiret** [GreenleafLab/ArchR#1993] [Feature Requests]
I have generated a browser track to display ATAC-seq peaks around a gene of interest (and a custom GRanges track) in only a subset of groups (clusters) using useGroups, and then added the GeneIntegrationMatrix panel using useMatrix = "GeneIntegrationMatrix". The resulting plot is attached.  In the GeneIntegrationMatrix panel, all of the clusters are displayed, instead of only the ones I selected for the browser track.  Is there any way to limit the GeneIntegrationMatrix panel to display the same 6 clusters as the browser track (and preferably in the same order also)?
Code here: 
`p <- plotBrowserTrack(
  ArchRProj = Klf6_Proj5, 
  groupBy = "predictedGroup", 
  useGroups = c("PT-S1-S2-1", "PT-S1-S2-2", "PT-S3-1", "PT-S3-2", "InjuredPT-A", "InjuredPT-B"),
  geneSymbol = "Polr2a",
  useMatrix = "GeneIntegrationMatrix",
  features =  gr_obj,
  upstream = 10000,
  downstream = 8000,
  ylim = c(0,1)
)
`
Thanks!
[BrowserTrack Polr2a PT KLF6_BS withGeneIntegration.pdf](https://github.com/GreenleafLab/ArchR/files/12099458/BrowserTrack.Polr2a.PT.KLF6_BS.withGeneIntegration.pdf)

**Comment by rcorces** [comment]
I dont currently have bandwidth to implement this but it shouldnt be that hard to do. I'm happy to review a pull request if someone wants to implement it.
